### PR TITLE
feat: update trending APR 23

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: build/universal/trending/english.yaml [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+  article25title: "Remove item from array JS"
+  article25link: "https://www.freecodecamp.org/news/how-to-remove-an-element-from-a-javascript-array-removing-a-specific-item-in-js/"
+  article26title: "Dual boot Windows + Ubuntu"
+  article26link: "https://www.freecodecamp.org/news/how-to-dual-boot-windows-10-and-ubuntu-linux-dual-booting-tutorial/"
+  article27title: "Python round to 2 decimals"
+  article27link: "https://www.freecodecamp.org/news/how-to-round-to-2-decimal-places-in-python/"
+  article28title: "String to Int in JavaScript"
+  article28link: "https://www.freecodecamp.org/news/string-to-number-in-javascript-convert-a-string-to-an-int-in-js/"
+  article29title: "What’s the .gitignore file?"
+  article29link: "https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/"
```
Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:60]:
```suggestion
article0title: "What is a framework?"
article0link: "https://www.freecodecamp.org/news/what-is-a-framework-software-frameworks-definition/"
article1title: "What do CS Majors Do?"
article1link: "https://www.freecodecamp.org/news/what-is-a-computer-scientist-what-exactly-do-cs-majors-do/"
article2title: "Discord update failed"
article2link: "https://www.freecodecamp.org/news/discord-update-failed-how-to-fix-the-error-on-a-windows-10-pc/"
article3title: "Center an Image in CSS"
article3link: "https://www.freecodecamp.org/news/how-to-center-an-image-in-a-div-css/"
article4title: "What is the MVC Model?"
article4link: "https://www.freecodecamp.org/news/what-does-mvc-mean-in-computer-science/"
article5title: "JavaScript replaceAll()"
article5link: "https://www.freecodecamp.org/news/javascript-replaceall-replace-all-instances-of-a-string-in-js/"
article6title: "Python switch statement"
article6link: "https://www.freecodecamp.org/news/python-switch-statement-switch-case-example/"
article7title: "Python string.replace()"
article7link: "https://www.freecodecamp.org/news/python-string-replace-how-to-replace-a-character-in-a-string/"
article8title: "What is a relational DB?"
article8link: "https://www.freecodecamp.org/news/what-is-a-relational-database-rdbms-definition/"
article9title: "Split a string in Python"
article9link: "https://www.freecodecamp.org/news/how-to-split-a-string-in-python/"
article10title: "SQL HAVING"
article10link: "https://www.freecodecamp.org/news/sql-having-how-to-group-and-count-with-a-having-statement/"
article11title: "What is OOP?"
article11link: "https://www.freecodecamp.org/news/what-is-object-oriented-programming/"
article12title: "HTML TextArea"
article12link: "https://www.freecodecamp.org/news/html-textarea-how-to-add-text-box-to-your-website/"
article13title: "NVM for Windows"
article13link: "https://www.freecodecamp.org/news/nvm-for-windows-how-to-download-and-install-node-version-manager-in-windows-10/"
article14title: "Git Revert File"
article14link: "https://www.freecodecamp.org/news/git-revert-file-reverting-a-file-to-a-previous-commit/"
article15title: "Group by in SQL"
article15link: "https://www.freecodecamp.org/news/sql-aggregate-functions-how-to-group-by-in-mysql-and-postgresql/"
article16title: "2D Array in Java"
article16link: "https://www.freecodecamp.org/news/2d-array-in-java-two-dimensional-and-nested-arrays/"
article17title: "How to install nvm"
article17link: "https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/"
article18title: "Percentages in Excel"
article18link: "https://www.freecodecamp.org/news/how-to-calculate-percentage-in-excel-formula-for-percentages/"
article19title: "JavaScript Timestamp"
article19link: "https://www.freecodecamp.org/news/javascript-timestamp-how-to-use-gettime-to-generate-timestamps-in-js/"
article20title: "Git list remote branches"
article20link: "https://www.freecodecamp.org/news/git-list-remote-branches/"
article21title: "Git Delete Remote Branch"
article21link: "https://www.freecodecamp.org/news/git-delete-remote-branch/"
article22title: "Software Developer Career"
article22link: "https://www.freecodecamp.org/news/what-does-a-software-developer-do-software-engineer-career-overview/"
article23title: "Three dots operator in JS"
article23link: "https://www.freecodecamp.org/news/three-dots-operator-in-javascript/"
article24title: "How to format dates in JS"
article24link: "https://www.freecodecamp.org/news/javascript-date-format-how-to-format-a-date-in-js/"
article25title: "Remove item from array JS"
article25link: "https://www.freecodecamp.org/news/how-to-remove-an-element-from-a-javascript-array-removing-a-specific-item-in-js/"
article26title: "Dual boot Windows + Ubuntu"
article26link: "https://www.freecodecamp.org/news/how-to-dual-boot-windows-10-and-ubuntu-linux-dual-booting-tutorial/"
article27title: "Python round to 2 decimals"
article27link: "https://www.freecodecamp.org/news/how-to-round-to-2-decimal-places-in-python/"
article28title: "String to Int in JavaScript"
article28link: "https://www.freecodecamp.org/news/string-to-number-in-javascript-convert-a-string-to-an-int-in-js/"
article29title: "What’s the .gitignore file?"
article29link: "https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/"
```

**Review by naomi-lgbt — APPROVED** [review]
image

Looks good to me. All links seem correct and titles render cleanly.

File: build/universal/trending/english.yaml [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
+article21link: "https://www.freecodecamp.org/news/git-delete-remote-branch/"
+article22title: "Software Developer Career"
+article22link: "https://www.freecodecamp.org/news/what-does-a-software-developer-do-software-engineer-career-overview/"
+article23title: "Three dots operator in JS"
+article23link: "https://www.freecodecamp.org/news/three-dots-operator-in-javascript/"
+article24title: "How to format dates in JS"
+article24link: "https://www.freecodecamp.org/news/javascript-date-format-how-to-format-a-date-in-js/"
+article25title: "Remove item from array JS"
+article25link: "https://www.freecodecamp.org/news/how-to-remove-an-element-from-a-javascript-array-removing-a-specific-item-in-js/"
+article26title: "Dual boot Windows + Ubuntu"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:53]:
```suggestion
article26title: "Dual Boot Windows + Ubuntu"
```

File: build/universal/trending/english.yaml [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
+article26link: "https://www.freecodecamp.org/news/how-to-dual-boot-windows-10-and-ubuntu-linux-dual-booting-tutorial/"
+article27title: "Python round to 2 decimals"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:55]:
```suggestion
article27title: "Python Round to 2 Decimals"
```

File: build/universal/trending/english.yaml [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
+article27link: "https://www.freecodecamp.org/news/how-to-round-to-2-decimal-places-in-python/"
+article28title: "String to Int in JavaScript"
+article28link: "https://www.freecodecamp.org/news/string-to-number-in-javascript-convert-a-string-to-an-int-in-js/"
+article29title: "What’s the .gitignore file?"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:59]:
```suggestion
article29title: "What’s the .gitignore File?"
```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Checked all the links and everything LGTM. They all seem to be resolve to the correct articles.

I did notice some inconsistencies with the capitalization for the article titles, and left some suggestions. They're minor details, but will be more consistent with the titles on News.

**Pull request merged**